### ISSUE TITLE
Manage Multiple Environments - variables

### ISSUE THREAD

**Issue by filippobarcellos** [infinitered/ignite#2439]
I noticed that the cookbook brings a recipe allowing us to set up env variables but I'm not sure how I would place them.

I need something like this:
- staging: API_URL: http://www.test.com/staging
- production: API_URL: http://www.test.com/production
- development: API_URL: http://www.test.com/development

Should I place to my babel.config.js like this:

```const expoConfig = {
  presets: ["babel-preset-expo"],
  env: {
    production: {
      API_URL: http://www.test.com/production,
   },
  staging: {
      API_URL: http://www.test.com/staging,
   },
  },
  plugins,
};
```

Thanks for your help.

**Comment by frankcalise** [comment]
Hey @filippobarcellos, thanks for your interest in Ignite! You can achieve this via the `app/config/*` config files. I think this fits your use case perfectly.

https://github.com/infinitered/ignite/blob/master/boilerplate/app/config/config.dev.ts

**Comment by RizqiSyahrendra** [comment]
> Hey @filippobarcellos, thanks for your interest in Ignite! You can achieve this via the `app/config/*` config files. I think this fits your use case perfectly.
> 
> https://github.com/infinitered/ignite/blob/master/boilerplate/app/config/config.dev.ts

But it's only two env (dev & release), how to set more env?

**Comment by vantuan88291** [comment]
> > Hey @filippobarcellos, thanks for your interest in Ignite! You can achieve this via the `app/config/*` config files. I think this fits your use case perfectly.
> > https://github.com/infinitered/ignite/blob/master/boilerplate/app/config/config.dev.ts
> 
> But it's only two env (dev & release), how to set more env?

you can config env inside native part like build.gradle on android and setup build config in xcode ios

**Comment by RizqiSyahrendra** [comment]
> > > Hey @filippobarcellos, thanks for your interest in Ignite! You can achieve this via the `app/config/*` config files. I think this fits your use case perfectly.
> > > https://github.com/infinitered/ignite/blob/master/boilerplate/app/config/config.dev.ts
> > 
> > 
> > But it's only two env (dev & release), how to set more env?
> 
> you can config env inside native part like build.gradle on android and setup build config in xcode ios

Isn't it different? I can call any config inside app/config/* from js code, but can I call it from js code if I use env on native part like build.gradle?

**Comment by vantuan88291** [comment]
> > > > Hey @filippobarcellos, thanks for your interest in Ignite! You can achieve this via the `app/config/*` config files. I think this fits your use case perfectly.
> > > > https://github.com/infinitered/ignite/blob/master/boilerplate/app/config/config.dev.ts
> > > 
> > > 
> > > But it's only two env (dev & release), how to set more env?
> > 
> > 
> > you can config env inside native part like build.gradle on android and setup build config in xcode ios
> 
> Isn't it different? I can call any config inside app/config/* from js code, but can I call it from js code if I use env on native part like build.gradle?

Yes its different, when you build the app you need to choose the env in native side, android use need select flavor, ios you need select configuration
